python/cudf/cudf/tests/utils.py

```diff
@@ -103,4 +103,4 @@ def gen_rand_series(dtype, size, **kwargs):
     if kwargs.get("has_nulls", False):
         return Series(values)
 
-    return Series.from_masked_array(values, utils.random_bitmask(size))
```

Comment:
This is a bug introduced in #2422. I will fix the calling code in a separate PR.